index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+    <html lang="en">
+        <head>
+            <meta charset="UTF-8">
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Sandeep's Portfolio</title>
+            <link rel="stylesheet" type="text/css" href="style.css">
+        </head>
+        <body>
+            <!-- navbar section -->
+            <section>
+                <header class="navbar">
+                    <a href="#">
+                        <h4>Sandeep</h4>
+                    </a>
+                    <nav>
+                        <div id="bar">
+                            <img src="images/Menu.png" alt="icon">
+                            <img src="images/Menu.png" alt="icon">
+                            <img src="images/Menu.png" alt="icon">
+                        </div>
```

Comment:
- [ ] [OPTIONAL] Using the `id` selector has no problem, however, for this div, we should consider using class selectors. You should use a class if you need to use the same selector more than once within a page or a site. While an ID is specific to a single element, classes can be assigned to multiple elements on a page or throughout the website ☺️.